extensions/integration-tests/dockerContainers.ps1

```diff
@@ -0,0 +1,21 @@
+# Get where docker is installed
+$dockerPath = (where.exe docker) | Select-Object -First 1
+$dockerDirectory = $dockerPath.Substring(0, $dockerPath.Length - 21)
+
+Write-Output "Waiting for Docker Desktop to start..."
+Start-Process -FilePath "$dockerDirectory\Docker Desktop.exe"
```

Comment:
I don't think this script is going to work on mac or linux for a few different reasons:

* Only Windows supports `\` for slash direction
* binaries on mac and linux won't be `.exe`

We can check this in, but then you should probably have a follow-up PR that makes sure this also works on mac.

Reply:
Working on this!

Reply:
done!